### DISCUSSION TITLE
Add opentofu plugin

### DISCUSSION THREAD

**Discussion by remiflament** [asdf-community/.github#64] [General]
Hello,

The first alpha of OpenToFu (Terraform fork) is out : https://github.com/opentofu/opentofu/releases/tag/v1.6.0-alpha1
Do we already have work in progress to add it support to ASDF?
Regards

**Comment by smorimoto** [comment]
I don't see any work going on for it.

**Comment by remiflament** [comment]
Hi, finally somebody did it! 🥳 https://github.com/virtualroot/asdf-opentofu
